### PR TITLE
feat: update cli-platform-android for find android dir on current directory

### PR DESCRIPTION
**Pull request by lynnkoo**
I created a project by exists android project, I think that andoid-cli can find current dirctory likes ios-cli.

### CODE REVIEW

File: packages/cli-platform-android/src/config/findAndroidDir.ts [hunk 1]
```diff
@@ -14,5 +14,12 @@ export default function findAndroidDir(root: string) {
     return 'android';
   }
 
+  if (
+    fs.existsSync(path.join(root, 'build.gradle'))
```
Comment (cortinico) [inline packages/cli-platform-android/src/config/findAndroidDir.ts:18]: Why is this needed?

Comment (lynnkoo) [reply packages/cli-platform-android/src/config/findAndroidDir.ts:18]: When I created a android app but not used `npx react-native` script, added a `package.json` and `yarn install` only on the android root directory, then run script `npx react-native config` can't found `project.android` info.
In the iOS project, I apply the same operations,  It's can work.
image
image
The different is `sourceDir` property when get from default value.

Comment (lynnkoo) [reply packages/cli-platform-android/src/config/findAndroidDir.ts:18]: This changes is not necessary, I can create a `react-native.config.js` config fie in android root directory alos.
